code/modules/admin/verbs/modifyvariables.dm

```diff
@@ -1,6 +1,6 @@
 var/list/forbidden_varedit_object_types = list(
```

Comment:
Oh also thanks for reminding me this list doesn't fucking do anything
It is not referenced anywhere else in the entire codebase, even once